serverless-plugin-canary-deployments.js

```diff
@@ -166,7 +171,8 @@ class ServerlessCanaryDeployments {
   }
 
   getDeploymentSettingsFor(serverlessFunction) {
-    const globalSettings = _.cloneDeep(this.service.custom.deploymentSettings);
+
+    const globalSettings = _.cloneDeep(_.prop(this.service, 'custom', 'deploymentSettings'));
```

Comment:
Since we're using `lodash/fp`, this is not gonna work, as the object passed to `.prop` is expected as last argument and the path must be passed in dot notation. You can see an usage example [here](https://runkit.com/embed/rru3skpxnvsz)

Reply:
Aha gotcha, one sec!